web/src/pages/AboutPage/AboutPage.tsx

```diff
@@ -1,19 +1,44 @@
 import { Link, routes } from '@redwoodjs/router'
 import { MetaTags } from '@redwoodjs/web'
 
+import Person from './Person'
+
 const AboutPage = () => {
   return (
     <>
       <MetaTags title="About" description="About page" />
-
-      <h1>AboutPage</h1>
-      <p>
-        Find me in <code>./web/src/pages/AboutPage/AboutPage.tsx</code>
-      </p>
-      <p>
-        My default route is named <code>about</code>, link to me with `
-        <Link to={routes.about()}>About</Link>`
-      </p>
+      <div className="grid grid-cols-2 grid-rows-3 gap-4 pb-4">
+        <Person
+          name="Adriana"
+          quote="えーどりあな"
+          story="Insert story about yourself here"
+          imgSrc="https://media.licdn.com/dms/image/D4E03AQErJ1uyanV3Hg/profile-displayphoto-shrink_200_200/0/1685713399087?e=1692230400&v=beta&t=SEH4cQD6g_GpZU3_RdsGtfFJDSCo8-CH2diebHYmhmQ"
```

Comment:
I see you have used some place to upload the images, am I right? Will these images be up there for the coming half year and can we trust its reliability? It might be better to just put the images in the `web/public/images` (you might to create the images folder manually) as we have direct control over it.

Reply:
No, I just used the pictures from linkedin for now. But I will create the images folder!